root/utility/releaseGroupType.js

```diff
@@ -14,7 +14,7 @@ export default function releaseGroupType(
 ): string {
   const types = [];
   let id = releaseGroup.typeID;
-  if (id) {
+  if (id != null) {
```

Comment:
Just note that this can change behavior in cases where we use an `id` of 0, which might just be some hacky places in edit classes. (Which is why I do think we think avoid that if possible.)

Reply:
I specifically used this *because* I thought we might have cases where we hackishly use an ID of 0 and we'd still want to display the stuff, even if the ID is falsy.

Reply:
That can be true, though this case is a bit different since we also require the ID to be in `linkedEntities` - I don't think an ID of 0 would be in there 'cause we'd use a positive number for a fake ID.

The code here shouldn't be affected at least, I was just making a general point about how using 0 as a placeholder complicates things. :)